### DISCUSSION TITLE
Can Milvus be configured to only search indexed segments?

### DISCUSSION THREAD

**Discussion by wangzhigang1999** [milvus-io/milvus#31166] [Q&A and General discussion] [answered]
I have a collection where there are **a lot of inserts happening concurrently with searches**. 

The new segments are not yet indexed, causing the search speed to be particularly slow. 

I can accept incomplete search results, but cannot tolerate high search latency. 

Is it possible to specify that Milvus **searches only indexed segments** through parameters during searching?

**Comment by yhmo** [comment, marked as answer]
The search interface has a parameter "ignore_growing" to ignore growing segments during searching, read the doc: https://milvus.io/docs/search.md

For sealed segments, maybe some segments are indexed, the others are not. No parameter to ignore the non-indexed sealed segments.

**Reply by wangzhigang1999** [reply]
This is very useful, thank you.
